python/phylanx/ast/transformation.py

```diff
@@ -279,6 +289,11 @@ def _Call(self, a, allowreturn=False):
             args = [arg for arg in ast.iter_child_nodes(a)]
             if args[0].id == "print":
                 args[0].id = "cout"
+            if args[0].id == "xrange":
+                args[0].id = "range"
```

Comment:
We have removed support for `xrange` as it's not a thing in Python V3 any more.

Reply:
Will remove from commit. Thanks!